<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>685</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="title">
           <string>Bloque 1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Parámetros configurables</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Tiempo experimento</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QSpinBox" name="b1_T_control"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Temperatura de control</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="b1_t_exp"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Intensidad LED Rojo</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Intensidad LED Verde</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="b1_slider_led_rojo">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QSlider" name="b1_slider_led_verde">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Configuración Fotos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Bloque 2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Parámetros configurables</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Tiempo experimento</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QSpinBox" name="b2_T_control"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Temperatura de control</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="b2_t_exp"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Intensidad LED Rojo</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Intensidad LED Verde</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="b2_slider_led_rojo">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QSlider" name="b2_slider_led_verde">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_8">
                <property name="title">
                 <string>Configuración Fotos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Bloque 3</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Parámetros configurables</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Tiempo experimento</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QSpinBox" name="b3_T_control"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Temperatura de control</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="b3_t_exp"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Intensidad LED Rojo</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Intensidad LED Verde</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="b3_slider_led_rojo">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksBelow</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QSlider" name="b3_slider_led_verde">
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksBelow</enum>
                  </property>
                  <property name="tickInterval">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>Configuración Fotos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Bloque 4</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Parámetros configurables</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Tiempo experimento</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QSpinBox" name="b4_T_control"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Temperatura de control</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="b4_t_exp"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Intensidad LED Rojo</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Intensidad LED Verde</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="b4_slider_led_rojo">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QSlider" name="b4_slider_led_verde">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_10">
                <property name="title">
                 <string>Configuración Fotos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Bloque 5</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Parámetros configurables</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Tiempo experimento</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QSpinBox" name="b5_T_control"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>Temperatura de control</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="b5_t_exp"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Intensidad LED Rojo</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Intensidad LED Verde</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="b5_slider_led_rojo">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QSlider" name="b5_slider_led_verde">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_11">
                <property name="title">
                 <string>Configuración Fotos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Bloque 6</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>Parámetros configurables</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>Tiempo experimento</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QSpinBox" name="b6_T_control"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_22">
                  <property name="text">
                   <string>Temperatura de control</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="b6_t_exp"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>Intensidad LED Rojo</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>Intensidad LED Verde</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="b6_slider_led_rojo">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QSlider" name="b6_slider_led_verde">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_12">
                <property name="title">
                 <string>Configuración Fotos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_guardar">
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
